paasta_tools/cli/cmds/rerun.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import datetime
+
+from paasta_tools.cli.cmds.status import get_actual_deployments
+from paasta_tools.cli.cmds.status import get_planned_deployments
+from paasta_tools.cli.cmds.status import list_deployed_clusters
+from paasta_tools.cli.utils import execute_chronos_rerun_on_remote_master
+from paasta_tools.cli.utils import figure_out_service_name
+from paasta_tools.cli.utils import lazy_choices_completer
+from paasta_tools.cli.utils import list_instances
+from paasta_tools.cli.utils import list_services
+from paasta_tools.utils import list_clusters
+from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import SPACER
+
+
+EXECUTION_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
```

Comment:
we should move this to ``chronos_tools`` so that we can reuse in backend validation, too.